util/upload-cookbook.py

```diff
@@ -0,0 +1,241 @@
+import re
+from datetime import datetime
+
+import argparse
+import boto3
+from botocore.exceptions import ClientError
+
+_BACKUP_DIR = "backup"
```

Comment:
we can avoid this global variable since it is used in a single place

Reply:
In general I follow the rule of using explicit constants to simplify maintenance and for better readability